substrate/frame/revive/rpc/src/receipt_extractor.rs

```diff
@@ -139,6 +144,13 @@ impl ReceiptExtractor {
 		&self,
 		block: &SubstrateBlock,
 	) -> Result<Vec<(TransactionSigned, ReceiptInfo)>, ClientError> {
+		if let Some(earliest_receipt_block) = self.earliest_receipt_block {
+			if block.number() < earliest_receipt_block {
+				log::debug!(target: LOG_TARGET, "Block number {block_number} is less than earliest receipt block {earliest_receipt_block}. Skipping.", block_number = block.number(), earliest_receipt_block = earliest_receipt_block);
```

Comment:
Could that possibly generate a lot of logs when indexing? Should it be trace maybe?

Reply:
sure let's make it trace